api.planx.uk/modules/flows/validate/service.ts

```diff
@@ -23,7 +25,7 @@ type AlteredNode = {
 
 type ValidationResponse = {
   title: string;
-  status: "Pass" | "Fail" | "Not applicable";
+  status: "Pass" | "Fail" | "Warn" | "Not applicable";
```

Comment:
I was expecting to see this change also picked up in `docs.yaml`, but it looks like we've missed adding in the `validationChecks` property.

Happy for this to be picked up here, or in a follow up PR 👍

Reply:
Thanks for spotting this, Swagger docs now updated :white_check_mark: 

![Screenshot from 2024-07-08 12-22-07](https://github.com/theopensystemslab/planx-new/assets/5132349/79103300-ce5a-440c-90be-76d1e1a8c607)